executor/update.go

```diff
@@ -112,6 +114,12 @@ func (e *UpdateExec) canNotUpdate(handle types.Datum) bool {
 
 // Next implements the Executor Next interface.
 func (e *UpdateExec) Next(ctx context.Context, chk *chunk.Chunk) error {
+	if e.execStat != nil {
+		start := time.Now()
+		defer func() {
```

Comment:
it can be simplified to:
```go
defer e.execStat.Record(time.Now().Sub(start), chk.NumRows())
```

Reply:
does `update` statement support the `explain` statement?

Reply:
`update` support it and will execute `selectExec` part  and not modify records...but this line is brainless added code that can be removed too... :joy_cat: